rpcs3/rpcs3_app.cpp

```diff
@@ -2,6 +2,16 @@
 
 #include "Emu/System.h"
 
+#include "Emu/RSX/GSRender.h"
```

Comment:
Doesn't make sense to have them all here

Reply:
Technically you should only include the .h needed for the objects declared in the .h in said .h.
Also I moved them there because in the .h they were causing some compilation issues.

Reply:
@RipleyTom but those were only meant for get_gs_render in main_application.cpp

Reply:
I assumed you'd want them all because of get_gs_frame but alright, removed them all except GSRender.h